README.md

```diff
@@ -48,7 +64,9 @@ The following configuration properties are available:
 - **defaultHandlingOfReceivedMessages**:boolean 
 If true, all incoming TCP messages are assumed to be string-encoded JSON objects according to the format described below. Set to false if you wish to implement your own custom handling of incoming messages.
 - **maxWaitForReply**:integer
-The max number of milliseconds to wait for the Model RealTime application to reply to a request for sending or invoking an event to it. If the TCPServer doesn't get a confirmation from the application within the specified time limit, it will assume that the Model RealTime application failed to receive the event and return an error message. 
+The max number of milliseconds to wait for the Model RealTime application to reply to a request for sending or invoking an event to it. If the TCPServer doesn't get a confirmation from the application within the specified time limit, it will assume that the Model RealTime application failed to receive the event and return an error message.
+- **maxSupportedClient**:integer
+The maximum number of supported client. This configuration is not available on POCO based release while available in master branch. This configuration requires Model RealTime release 12.0.2 or later.
```

Comment:
client -> clients

No need to document the POCO version since that's already done in the README the user will see when he checks out the Poco tag.